contracts/RiskManagerV1.sol

```diff
@@ -142,6 +161,38 @@ contract RiskManagerV1 is Auctioneer, Ownable {
         earlyCloseAuction(auction);
     }
 
+    /// @notice Begins the collateralization threshold update process.
+    /// @dev Can be called only by the contract owner. The collateralization
+    ///      threshold should be adjusted by taking into account factors like
+    ///      collateralization ratio and gas price.
+    /// @param _collateralizationThreshold New collateralization threshold in percent.
+    function beginCollateralizationThresholdUpdate(
+        uint256 _collateralizationThreshold
```

Comment:
Maybe `_newCollateralizationThreshold` to stay consistent with `beginAuctionLengthUpdate`?

Reply:
81c3fe3